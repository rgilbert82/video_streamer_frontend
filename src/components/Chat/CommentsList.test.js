import React            from 'react';
import { shallow }      from 'enzyme';
import { CommentsList } from '.';

describe(CommentsList, () => {
  let component;

  beforeEach(() => {
    component = shallow(<CommentsList displayMessage={jest.fn()}/>);
  });

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
});
