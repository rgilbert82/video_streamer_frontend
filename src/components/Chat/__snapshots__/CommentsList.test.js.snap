// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class CommentsList extends _react2.default.Component {
  render() {
    let messages = this.props.messages.map(message => {
      return (
        _react2.default.createElement('li', { key: message.comment.id, className: 'commentListItem', __source: { fileName: _jsxFileName, lineNumber: 8 }, __self: this },
          _react2.default.createElement(_.CommentListItem, {
            message: message,
            loggedIn: this.props.loggedIn,
            currentUser: this.props.currentUser, __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: this })));



    });

    return (
      _react2.default.createElement('div', { id: 'videoCommentsList', __source: { fileName: _jsxFileName, lineNumber: 19 }, __self: this },
        _react2.default.createElement('ul', { __source: { fileName: _jsxFileName, lineNumber: 20 }, __self: this },
          messages)));



  }} renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CommentsList
    displayMessage={[Function]}
    messages={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul>
        
</ul>,
      "id": "videoCommentsList",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [],
      },
      "ref": null,
      "rendered": Array [],
      "type": "ul",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul>
          
</ul>,
        "id": "videoCommentsList",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": "ul",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
